docs/develop/go/index.mdx

```diff
@@ -57,9 +59,9 @@ The [Failure detection feature guide](/develop/go/failure-detection) shows how y
 
 The [Application messages feature guide](/develop/go/message-passing) shows how to send messages to and read the state of Workflow Executions.
 
-- [How to develop with Signals](/develop/go/message-passing#signals)
-- [How to develop with Queries](/develop/go/message-passing#queries)
-- [How to develop with Updates](/develop/go/message-passing#updates)
+- [Signals](/develop/go/message-passing#signals)
+- [Queries](/develop/go/message-passing#queries)
+- [Updates](/develop/go/message-passing#updates)
```

Comment:
I hate that these are single words. However if they align with other SDK coverage, there's nothing to do about it.